src/session.ts

```diff
@@ -98,32 +98,41 @@ export class DebugSession implements IDebugger.ISession {
    * Start a new debug session
    */
   async start(): Promise<void> {
-    this._isStarted = true;
-    await this.sendRequest('initialize', {
-      clientID: 'jupyterlab',
-      clientName: 'JupyterLab',
-      adapterID: 'python',
-      pathFormat: 'path',
-      linesStartAt1: true,
-      columnsStartAt1: true,
-      supportsVariableType: true,
-      supportsVariablePaging: true,
-      supportsRunInTerminalRequest: true,
-      locale: 'en-us'
-    });
-
-    await this.sendRequest('attach', {});
+    try {
+      await this.sendRequest('initialize', {
+        clientID: 'jupyterlab',
+        clientName: 'JupyterLab',
+        adapterID: 'python',
+        pathFormat: 'path',
+        linesStartAt1: true,
+        columnsStartAt1: true,
+        supportsVariableType: true,
+        supportsVariablePaging: true,
+        supportsRunInTerminalRequest: true,
+        locale: 'en-us'
+      });
+
+      this._isStarted = true;
+
+      await this.sendRequest('attach', {});
+    } catch (err) {
+      console.log('Error: ', err.message);
```

Comment:
What to you think about using `console.error` instead?

Reply:
Indeed that would be better. I'll update the PR